### PR TITLE
Tweak footer CSS

### PR DESCRIPTION
**Pull request by lpinca**
This fixes some inconsistencies between various browsers.

Here are some screenshots:

**IE11**

ie11

**Firefox (before)**

ff_before

**Firefox (after)** 

ff_after

I had to use a png image instead of the original one (svg inlined in the css) because the latter was indented with certain screen sizes.

I tested this on Chrome, Safari (desktop and mobile), Firefox and IE11.

### CODE REVIEW

File: layouts/css/layout/_sticky-footer.styl [hunk 1]
```diff
@@ -13,7 +13,8 @@ body
 #main
     flex 1 0 auto
 
-header
+header,
+footer
```
Comment (lpinca) [inline layouts/css/layout/_sticky-footer.styl:17]: Prevent some browsers from letting this item shrink to smaller than its content's default minimum size.

File: static/images/lfcp.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...-.423.152-.784.455-1.08s.674-.442 1.11-.442M100.072.08v2.968c0 .327-.062.553-.185.677s-.27.187-.437.187c-.04 0-.086-.003-.138-.008l-.022-.217c.048.01.094.017.136.017.278 0 .417-.198.417-.595V.08h.23zM102.472.08v.19h-1.39v1.19h1.354v.192h-1.353v1.204h1.423v.192h-1.653V.08h1.62zM105.472.29v.25c-.36-.19-.712-.286-1.058-.286-.368 0-.676.125-.924.376-.248.25-.372.564-.372.94 0 .378.125.69.374.936.25.245.567.367.954.367.323 0 .675-.1 1.056-.304v.237c-.35.186-.702.28-1.052.28-.45 0-.828-.146-1.13-.436-.3-.288-.45-.65-.45-1.08 0-.436.153-.8.46-1.09s.674-.44 1.1-.44c.333 0 .68.084 1.042.25M108.192.08v.19h-1.113V3.05h-.233V.27h-1.113V.08h2.458zM109.882.22v.253c-.258-.152-.48-.227-.665-.227-.178 0-.327.054-.447.162-.12.108-.18.243-.18.402 0 .115.036.22.107.31.07.093.218.18.444.266.226.085.39.156.493.213.104.055.192.137.264.245.072.108.108.247.108.42 0 .227-.083.42-.25.576s-.37.237-.612.237c-.247 0-.5-.086-.76-.26v-.27c.287.212.536.318.747.318.176 0 .324-.057.445-.17.12-.115.182-.255.182-.42 0-.127-.038-.24-.113-.342s-.227-.196-.456-.284-.394-.16-.492-.214c-.1-.053-.182-.13-.25-.23s-.104-.228-.104-.385c0-.217.084-.403.25-.557.168-.153.37-.23.604-.23.23 0 .46.062.686.187" fill="#309245"/></svg>
```
Comment (lpinca) [inline static/images/lfcp.svg:1]: This is the original svg image previously inlined in the CSS.

**Comment by diagramatics** [comment]
Looks great!

**Comment by hiroppy** [comment]
LGTM :+1:

**Comment by fhemberger** [comment]
Great, merged.

**Pull request merged**